dash_core_components/metadata.json

```diff
@@ -2519,5 +2519,192 @@
         "description": ""
       }
     }
+  },
+  "src/components/Upload.react.js": {
+    "description": "",
+    "methods": [],
+    "props": {
+      "id": {
+        "type": {
+          "name": "string"
+        },
+        "required": false,
+        "description": "ID of the component. Used to identify component\nin Dash callback functions."
+      },
+      "contents": {
```

Comment:
Would it be possible to add a "filename(s)" property here to indicated which file or files have been dropped?  Then set the property on line 35 of src/components/Upload.react.js, and add to Upload.propTypes?  Or is there a way to get the filename that I'm not seeing?

Thanks

Reply:
Good idea @kevinchiang ! Added this in https://github.com/plotly/dash-core-components/pull/73/commits/dfa51bdbc066e6a6d807a1dae177192db0bf334d

Reply:
Hi, the filename is a great feature. But it did not work for me so far. If I install this commit clean I have warning that bundle.js is not found. If I copy bundle.js from 0.13.0rc7 then It gives me None for both filename and date_modified. What might be the reason, thanks